MyCapytain/errors.py

```diff
@@ -64,3 +64,8 @@ class UnknownCollection(KeyError, MyCapytainException):
 class EmptyReference(SyntaxWarning, MyCapytainException):
     """ Error generated when a duplicate is found in Reference
     """
+
+
+class MissingRefsDecl(KeyError, MyCapytainException):
```

Comment:
Remove sublcassing of KeyError seems better here. I can't find a type of error that feels not overfitted.